src/env_loader.zig

```diff
@@ -431,6 +432,55 @@ pub const Loader = struct {
             this.map.get("bamboo.buildKey")) != null;
     }
 
+    pub fn getHttpProxy(this: *Loader, url: URL) ?URL {
+        // TODO: When Web Worker support is added, make sure to intern these strings
+        var http_proxy: ?URL = null;
+
+        if(url.isHTTP()){
+        if (this.map.get("http_proxy")) |proxy| {
+                http_proxy = URL.parse(proxy);
+            } else if (this.map.get("HTTP_PROXY")) |proxy| {
+                http_proxy = URL.parse(proxy);
+            }
+        }else{
+            if (this.map.get("https_proxy")) |proxy| {
```

Comment:
Instead of two identical if else branches, we can do

```suggestion
            if (this.map.get("https_proxy") orelse this.map.get("HTTPS_PROXY")) |proxy| {
```

Reply:
done